.changeset/khaki-countries-occur.md

```diff
@@ -0,0 +1,5 @@
+---
+'@igloo-ui/button': minor
```

Comment:
The change is major here, we replace the UI of the ghost variant which can cause regression errors.